model/project_configs.go

```diff
@@ -24,14 +24,19 @@ type ProjectConfig struct {
 	BuildBaronSettings     *evergreen.BuildBaronSettings  `yaml:"build_baron_settings,omitempty" bson:"build_baron_settings,omitempty"`
 	PerfEnabled            *bool                          `yaml:"perf_enabled,omitempty" bson:"perf_enabled,omitempty"`
 	CommitQueueAliases     []ProjectAlias                 `yaml:"commit_queue_aliases,omitempty" bson:"commit_queue_aliases,omitempty"`
-	GitHubPRAliases        []ProjectAlias                 `yaml:"github_pr_aliases,omitempty" bson:"github_pr_aliases,omitempty"`
+	GitHubPRAliases        []ProjectAlias                 `yaml:"github_aliases,omitempty" bson:"github_aliases,omitempty"`
```

Comment:
What's the thought behind making this github_aliases instead of github_pr_aliases? The latter is preferable since it fits convention with the Project struct and the variable names.

Reply:
I did this to be consistent with service/project.go: https://github.com/evergreen-ci/evergreen/blob/main/service/project.go#L280 although I've now renamed them both to github_pr_aliases